### DISCUSSION TITLE
White Label - Publish same app in two different accounts

### DISCUSSION THREAD

**Discussion by endritgojanikodelabs** [codemagic-ci-cd/codemagic-docs#1345] [Q&A] [answered]
Hello everyone, I'm doing White Label in one of my apps, and we sell the product like a service. And one of client wants that app to be publish from his account on Apple/Android. Is this possible in code magic

**Comment by mohammedbabelly20** [comment, marked as answer]
Hey @endritgojanikodelabs,

Yes it is possible.
I'm not sure how you are handling the white labeling scripts for each client, but the general way is to store each client's assets and secrets in a zipped file, and when the build starts for a specific client (you can pass the client Id via REST API) you download the zipped file for this client and replace all the assets and export the client's keys, certificates, and provisioning profile that will Codemagic use in the publishing section.

You can read more on white labeling [here](https://docs.codemagic.io/knowledge-white-label/white-label-scripts/).
